sbfl/diagnosability.py

```diff
@@ -2,11 +2,11 @@
 
 def diversity(X):
     N, _ = X.shape
+    if N < 2:
```

Comment:
How about `N == 1` ? 😛

Reply:
Oh! I worried following case

```
>>> x = np.array([])
>>> x.shape
(0, )
```

But I found that unpacking error occurs in the case 😄 
```
>>> x = np.array([])
>>> N, M = x.shape    # Value error here
```

Thus, no problem at all for `N == 1`
Thanks a lot

Reply:
@agb94 All done :ok_hand: